hack/ci/e2e-k8s.sh

```diff
@@ -122,9 +135,9 @@ apiVersion: kind.x-k8s.io/v1alpha4
 networking:
   ipFamily: ${IP_FAMILY:-ipv4}
 nodes:
-- role: control-plane
-- role: worker
-- role: worker
+- role: control-plane $(add_extra_mounts "$EXTRA_MOUNTS")
```

Comment:
More of a question: would it be worth it to have different mounts for control-plane and worker notes?

Reply:
Hm, maybe for even more enhanced testing. But this would increase the complexity. I just thought that we statically mount only one seccomp test-profile dir for now.